include/scrimmage/plugins/interaction/GraphInteraction/GraphInteraction.h

```diff
@@ -41,6 +41,12 @@
 #include <map>
 #include <list>
 #include <string>
+#include <vector>
+#include <deque>
```

Comment:
is this used anywhere? It  looks like it should be removed. We can also forward declare Entity, Message, and Publisher.